cmd/pi-strings/src/main.rs

```diff
@@ -132,6 +132,8 @@ fn extract_strings_from_path(filename: String) -> Result<Vec<strings::StringInfo
                 let mut sinfo = strings::StringInfo::default();
                 sinfo.set_location((index - s.len()) as u64);
                 sinfo.set_raw_string(s.to_string());
+                sinfo.set_size(s.len() as u64);
```

Comment:
Interesting that rust named `String::len` for number of bytes, I would have named it `String::size`

Reply:
I'm a Rust noob, so may very well be that there also exist a `size` method. Just made the minimal change to make it compile, and seeing that it was ASCII input then size=len :p